package.json

```diff
@@ -29,11 +30,11 @@
   },
   "dependencies": {
     "@codemirror/next": "^0.12.0",
-    "@material/mwc-button": "^0.18.0",
-    "@material/mwc-icon-button": "^0.18.0",
-    "@material/mwc-tab": "^0.18.0",
-    "@material/mwc-tab-bar": "^0.18.0",
-    "@material/mwc-textfield": "^0.18.0",
+    "@material/mwc-button": "^0.19.0-canary.615d861d.0",
```

Comment:
with the hashes these might as well be exact versions

Reply:
Done.